packages/flutter/lib/src/material/chip.dart

```diff
@@ -1806,7 +1825,7 @@ class _RawChipState extends State<RawChip> with MaterialStateMixin, TickerProvid
           onTap: widget.isEnabled ? widget.onDeleted : null,
           child: IconTheme(
             data: theme.iconTheme.copyWith(
-              color: widget.deleteIconColor ?? chipTheme.deleteIconColor,
+              color: widget.deleteIconColor ?? chipTheme.deleteIconColor ?? theme.chipTheme.deleteIconColor ?? chipDefaults.deleteIconColor,
```

Comment:
Maybe wrap these like above: seems kind of long.

Reply:
Done.